### DISCUSSION TITLE
Does PyG has the implementation on Graph2Seq model?

### DISCUSSION THREAD

**Discussion by haoyuan-l** [pyg-team/pytorch_geometric#5916] [Q&A]
Hi, I have been working on implementing multiple GNN models with PyG. I noticed that the `**Graph2Seq**` model is not contained in the cheatsheet. I would like to use PyG for all the models to make the project more consistent. Is there any related model/implementations I  omitted?  Any suggestions will be appreciated!

Here is the original paper for the model:
[Graph2Seq](https://arxiv.org/abs/1804.00823)

**Comment by wsad1** [comment]
We don't have `Graph2Seq` in pyg. But it'd be nice to add it. Let us know if you are interested in contributing.

**Reply by haoyuan-l** [reply]
Hi, I would like to contribute to PyG community! But currently, I prefer to use the GGNN module within the package to save time (Both models have the sequence decoder I need for my project). If I got time after I implement the GGNN on my project. I will try to implement Graph2Seq with PyG. Thank you again for this fantastic package!
